README.md

```diff
@@ -0,0 +1,112 @@
+# Kokkos Documentation Workflow <br/><br/>
+
+Here, we summarize the steps for setting up the kokkos-core-wiki repo, and a
+branch-based workflow for updating documentation in the main Kokkos project.  The kokkos-core-wiki is the
+canonical source of Kokkos documentation "Truth".<br/>
+
+Sandia employees: Creation of wholly new pages will require Review and Approval (R&A).  Check with your Sandia team lead, if you have questions about the R&A process.<br/><br/>
+
+* **Step 1) Create your own fork of the kokkos-core-wiki repo by navigating to `https://github.com/kokkos/kokkos-core-wiki`, and clicking "Fork" in the upper right of the github page**.<br/>
+
+* **Step 2) Clone your forked repo, and add it as a remote:**<br/>
+
+```
+git clone git@github.com:ajpowelsnl/kokkos-core-wiki.git
+git remote add ajpowelsnl
+git@github.com:ajpowelsnl/kokkos-core-wiki.git
+```
+
+* **Step 3) Add the remote repo as "origin" or "upstream", according to your preference.  You will create pull requests against this repo:**<br/>
+
+```
+git remote add origin git@github.com:kokkos/kokkos-core-wiki.git
+```
+
+* **Step 4) Check your remotes; you should see both your fork (`ajpowelsnl`) and the main project repo (`origin`):**<br/>
+
+```
+[ajpowel@kokkos-dev-2 kokkos-core-wiki]$ git remote -v
+ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git (fetch)
+ajpowelsnl git@github.com:ajpowelsnl/kokkos-core-wiki.git (push)
+origin https://github.com/kokkos/kokkos-core-wiki.git (fetch)
+origin https://github.com/kokkos/kokkos-core-wiki.git (push)
+```
+
+* **Step 5) Create a topic branch in your local fork of the kokkos-core-wiki:**<br/>
+
+```
+git checkout -b demo/setup
+```
+
+* **Step 6) Add the Kokkos project Wiki repo from your fork as a remote (you will push changes to this repo to preview before creating a pull request on `origin git@github.com:kokkos/kokkos-core-wiki.git`):**<br/>
+
+```
+git remote add my_wiki git@github.com:ajpowelsnl/kokkos.wiki.git
+```
+
```

Comment:
Before doing this, you should make sure that the `Wiki` is not empty. At least, I couldn't push to or pull from this repository before creating a dummy page.